src/client/store/user/hooks/index.ts

```diff
@@ -18,6 +19,16 @@ export const useUserCountries = (): Array<CountryIso> => {
   return user?.roles.filter((role) => cycle.uuid === role.cycleUuid).map((role) => role.countryIso)
 }
 
+export const useUserCycles = (assessment: Assessment): Array<Cycle> => {
+  const user = useUser()
```

Comment:
You can avoid passing assessment here , you can use useAssessment same way you use useUser